libs/VideoKit/source/VideoKit.cpp

```diff
@@ -48,6 +48,7 @@ void VideoKit::playVideo(const std::filesystem::path &path, bool must_loop)
 		_event_flags.set(flags::START_VIDEO_FLAG);
 
 		_lcd.turnOn();
+		_on_video_ended = onVideoDidEndCallback;
```

Comment:
```suggestion
		_on_video_ended = onVideoEndedCallback;
```

ou si tu veux revenir comme avant

```suggestion
		_on_video_did_end = onVideoDidEndCallback;
```